### ISSUE TITLE
ImportError: cannot import name 'abs'

### ISSUE THREAD

**Issue by rlan** [rlan/notebooks#27]
```sh
nvidia-docker run -it --rm wqael/notebooks:keras-tf1.4.1-conda3-cuda8 bash
```

```
>>> import tensorflow as tf
  File "/opt/conda/lib/python3.6/site-packages/tensorflow/python/layers/base.py", line 25, in <module>
    from tensorflow.python.keras.engine import base_layer
  File "/opt/conda/lib/python3.6/site-packages/tensorflow/python/keras/engine/__init__.py", line 23, in <module>
    from tensorflow.python.keras.engine.base_layer import InputSpec
  File "/opt/conda/lib/python3.6/site-packages/tensorflow/python/keras/engine/base_layer.py", line 35, in <module>
    from tensorflow.python.keras import backend
  File "/opt/conda/lib/python3.6/site-packages/tensorflow/python/keras/backend/__init__.py", line 22, in <module>
    from tensorflow.python.keras._impl.keras.backend import abs
ImportError: cannot import name 'abs'
```

```sh
nvidia-docker run -it --rm wqael/notebooks:keras-tf1.4.1-conda2-cuda8 bash
```

```
>>> import tensorflow as tf
  File "/opt/conda/lib/python2.7/site-packages/tensorflow/__init__.py", line 30, in <module>
    from tensorflow.python import pywrap_tensorflow  # pylint: disable=unused-import
  File "/opt/conda/lib/python2.7/site-packages/tensorflow/python/__init__.py", line 88, in <module>
    from tensorflow.python import keras
  File "/opt/conda/lib/python2.7/site-packages/tensorflow/python/keras/__init__.py", line 24, in <module>
    from tensorflow.python.keras import activations
  File "/opt/conda/lib/python2.7/site-packages/tensorflow/python/keras/activations/__init__.py", line 22, in <module>
    from tensorflow.python.keras._impl.keras.activations import elu
  File "/opt/conda/lib/python2.7/site-packages/tensorflow/python/keras/_impl/keras/__init__.py", line 21, in <module>
    from tensorflow.python.keras._impl.keras import activations
  File "/opt/conda/lib/python2.7/site-packages/tensorflow/python/keras/_impl/keras/activations.py", line 23, in <module>
    from tensorflow.python.keras._impl.keras import backend as K
  File "/opt/conda/lib/python2.7/site-packages/tensorflow/python/keras/_impl/keras/backend.py", line 36, in <module>
    from tensorflow.python.layers import base as tf_base_layers
  File "/opt/conda/lib/python2.7/site-packages/tensorflow/python/layers/base.py", line 25, in <module>
    from tensorflow.python.keras.engine import base_layer
  File "/opt/conda/lib/python2.7/site-packages/tensorflow/python/keras/engine/__init__.py", line 23, in <module>
    from tensorflow.python.keras.engine.base_layer import InputSpec
  File "/opt/conda/lib/python2.7/site-packages/tensorflow/python/keras/engine/base_layer.py", line 35, in <module>
    from tensorflow.python.keras import backend
  File "/opt/conda/lib/python2.7/site-packages/tensorflow/python/keras/backend/__init__.py", line 22, in <module>
    from tensorflow.python.keras._impl.keras.backend import abs
ImportError: cannot import name abs
```

Blocks #21

**Comment by rlan** [comment]
```
nvidia-docker run -it --rm wqael/notebooks:keras-tf1.4.1-conda3-cuda8 bash
```

```
root@4855e888e328:/# conda list | grep tensorflow
tensorflow                1.12.0          mkl_py36h69b6ba0_0
tensorflow-base           1.12.0          mkl_py36h3c3e929_0
tensorflow-gpu            1.4.1                         0
tensorflow-gpu-base       1.4.1            py36h01caf0a_0
tensorflow-tensorboard    1.5.1            py36hf484d3e_1
```

**Comment by rlan** [comment]
> ```
> nvidia-docker run -it --rm wqael/notebooks:keras-tf1.4.1-conda3-cuda8 bash
> ```
> Problem:
> 
> ```
> root@4855e888e328:/# conda list | grep tensorflow
> tensorflow                1.12.0          mkl_py36h69b6ba0_0
> tensorflow-base           1.12.0          mkl_py36h3c3e929_0
> tensorflow-gpu            1.4.1                         0
> tensorflow-gpu-base       1.4.1            py36h01caf0a_0
> tensorflow-tensorboard    1.5.1            py36hf484d3e_1
> ```

Fix:

```
      keras-gpu \
      tensorflow-gpu=1.4.1 \
```

**Comment by rlan** [comment]
Fixed because #21 is fixed.
